django_react_proj/users-fe/src/components/ConfirmRemovalModal.js

```diff
@@ -1,53 +1,56 @@
-import React, { Component, Fragment } from "react";
+import React, { useState, Fragment } from "react";
 import { Modal, ModalHeader, Button, ModalFooter } from "reactstrap";
 import axios from "axios";
 import { API_URL } from "../constants";
 
-class ConfirmRemovalModal extends Component {
-  state = {
-    modal: false
-  };
+const ConfirmRemovalModal = (props) => {
+  const [modalFlag, setModalFlag] = useState(
+        false
+  )
 
-  toggle = () => {
-    this.setState(previous => ({
-      modal: !previous.modal
-    }));
+  const toggle = () => {
+    setModalFlag(previous => (
+       !previous
+     ));
   };
+  
+  const handlerDeleteUser = async id => {
+    try {
+      await props.deleteUser(id)
+      toggle() 
+    } catch (error) {
+      console.log(error)
+    } 
 
-  deleteUser = id => {
-    axios.delete(API_URL + id).then(() => {
-      this.props.resetState();
-      this.toggle();
-    });
-  };
+   };
+  //gppp
+
+  return (
+    <Fragment>
```

Comment:
You can use just `<></>` instead of `<Fragment></Fragment>` (it's just a shortcut for Fragment)